### PR TITLE
Add 0.3.0-ballot to package-list (https://github.com/hl7au/au-fhir-core/issues/175)

### LINKED ISSUE #175: au core package-list.json & history.html files are out of date
The package-list file & history.html file in this repository is missing 0.3.0-ballot

### PR DESCRIPTION
**Pull request by dbojicic**
Fixes #175. 

Add 0.3.0-ballot to package-list. History file is generated from the package-list and does not require additional changes.

### CODE REVIEW

File: package-list.json [hunk 1]
```diff
@@ -38,7 +47,7 @@
       "path": "http://hl7.org.au/fhir/core/0.2.0-preview",
       "fhirversion": "4.0.1",
       "status": "draft",
-      "sequence": "AU Core R4"
```
Comment (dt-r) [inline package-list.json:41]: This change is not related to adding 0.3.0-ballot, it is changing how sequencing is categorised. The current sequence approach was indicating the version of FHIR not the release of AU Core. We will need to meet on this to change how the versioning is progressing and discuss AU Base and other HL7 AU IGs.

File: package-list.json [hunk 2]
```diff
@@ -13,14 +13,23 @@
...
       "current": true
     },
+    {
+      "version": "0.3.0-ballot",
+      "date": "2024-03-10",
+      "desc": "This snapshot is published for ballot and the March 2024 Connectathon.",
+      "path": "https://hl7.org.au/fhir/core/0.3.0-ballot",
+      "fhirversion": "4.0.1",
+      "status": "draft",
+      "sequence": "AU Core R1"
```
Comment (dt-r) [inline package-list.json:23]: Again, sequence convention change requires a meeting to change.

File: package-list.json [hunk 3]
```diff
@@ -13,14 +13,23 @@
       "fhirversion": "4.0.1",
       "current": true
     },
+    {
+      "version": "0.3.0-ballot",
+      "date": "2024-03-10",
+      "desc": "This snapshot is published for ballot and the March 2024 Connectathon.",
```
Comment (dt-r) [inline package-list.json:19]: The desc is not sufficient, but since it matches what is already in the history it must be kept.

Going forwards enough information to know what the snapshot is for, e.g. Ballot for Comment etc. needs to be provided. Preferably 2-3 sentences.

You may resolve this conversation, but ensure that the testing snapshot is released with a meaningful desc.

File: package-list.json [hunk 4]
```diff
@@ -13,14 +13,23 @@
       "fhirversion": "4.0.1",
       "current": true
     },
```
Comment (dt-r) [inline package-list.json:15]: Overall, is this file actually maintained in GitHub? I can't tell since the 0.3.0-ballot has already been published without this file being changed. Where is the source of truth?

**Review by dt-r — CHANGES_REQUESTED** [review]
This PR leaves me with questions on history vs package-list.

**Comment by dt-r** [comment]
One last question. On the comment that the history.html file is automatically generated. Are you saying that on merging of this PR the following file will be automatically updated": https://github.com/hl7au/au-fhir-core/blob/master/history.html

File: package-list.json (hunk 3 above)
Comment (dbojicic) [reply package-list.json:19]: noted

**Comment by dbojicic** [comment]
> One last question. On the comment that the history.html file is automatically generated. Are you saying that on merging of this PR the following file will be automatically updated?: https://github.com/hl7au/au-fhir-core/blob/master/history.html

The history file is produced as an output of -go-publish and does not need to be constructed and/or added manually to the root repo anymore. Removed an outdated history file from this repo.

**Review by dt-r — APPROVED** [review]
